gigareview/common/models/Manuscript.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace common\models;
+
+use Yii;
+use yii\behaviors\TimestampBehavior;
+
+/**
+ * This is the model class for table "manuscript".
+ *
+ * @property int $id
+ * @property string|null $manuscript_number
+ * @property string|null $article_title
+ * @property int|null $revision_number
+ * @property int|null $created_at
+ * @property int|null $updated_at
+ */
+class Manuscript extends \yii\db\ActiveRecord
```

Comment:
This function needs to be updated according to changes required in `m220629_145450_create_manuscript_table` below.